modules/ROOT/partials/configuration/text_color.adoc

```diff
@@ -23,10 +23,68 @@ tinymce.init({
 });
 ----
 
+[[color_cols_background]]
+=== `color_cols_background`
+
+This option specifies the number of columns for background text color grids.
+
+If `color_cols_background` is not set, the number of columns a background text color grid takes is set by the xref:#color_cols[`color_cols`] option.
+
+If both `color_cols` and `color_cols_background` options are set, the `color_cols_background` value is used.
+
+If neither `color_cols` nor `color_cols_background` options are set, background text color grids present with the default number of columns, **5**.
+
+*Type:* `+Integer+`
+
+*Default value:* 5
+
+==== Example: using `color_cols_background`
+
+This example sets the number of text color grid columns to **10** then sets the background text color grid to display **3** columns.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  toolbar: 'backcolor forecolor',
+  color_cols: 10,
+  color_cols_background: 3,
+});
+----
+
+[[color_cols_foreground]]
+=== `color_cols_foreground`
+
+This option specifies the number of columns for foreground text color grids.
+
+If `color_cols_foreground` is not set, the number of columns a background text color grid takes is set by the xref:#color_cols[`color_cols`] option.
+
+If both `color_cols` and `color_cols_foreground` options are set, the `color_cols_foreground` value is used.
+
+If neither `color_cols` nor `color_cols_foreground` options are set, background text color grids present with the default number of columns, **5**.
+
+*Type:* `+Integer+`
+
+*Default value:* 5
+
+==== Example: using `color_cols_foreground`
+
+This example sets the number of text color grid columns to **10** then sets the foreground text color grid to display **3** columns.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  toolbar: 'backcolor forecolor',
+  color_cols: 10,
+  color_cols_foreground: 3,
+});
+----
+
 [[color_map]]
 === `+color_map+`
 
-This option allows specifying a map of the text colors that will appear in the grid.
+This option allows for specifying a map of the text colors that will appear in the grid.
```

Comment:
```suggestion
This option allows for specifying a map of the text colors that will appear in the grid.

Colors are added to the array with a Hex or RGB color value, followed by the name of that color as it would appear in the color grid when hovered over. Both values are required for each color added to the color map.
```
Since I don't think this is described anywhere. Sounds like a weird method to me but that's what it looks like it expects.